src/pages/managerForm.js

```diff
@@ -1,66 +1,209 @@
-            <div className={styles.firstBanner}>New Community</div>
-            <div className={styles.secondBanner}>
-                <h2>Choose category(s)</h2>
-                <div className={styles.secondBannerButtons}>
-                    <CustomizableButton
-                        customClass={styles.disableButton}
-                        onClickProp={categoryChoice}
-                        text="UI/UX"
-                    />
-                    
-                    <CustomizableButton
-                        customClass={styles.disableButton}
-                        onClickProp={categoryChoice}
-                        text="Frontend"
-                    />
-                    
-                    <CustomizableButton
-                        customClass={styles.highlightButton}
-                        onClickProp={categoryChoice}
-                        text="Software Engineering"
-                    />
-
-                    <CustomizableButton
-                        customClass={styles.disableButton}
-                        onClickProp={categoryChoice}
-                        text="Product Design"
-                    />
-
-                    <Image
-                        src='/arrow-right-line.png'
-                        width={25}
-                        height={25}
-                        alt='arrow pointing to the right'
-                    />
-                </div>
-
-                <div className={styles.communityGrid}>
+  ];
+  const submitCommunity = async (event) => {
+    event.preventDefault();
+    const commCollectionRef = collection(
+      firebaseDb,
+      "MANAGERS",
+      "3fF17YaSFLHgRgksgVU8",
+      "MANAGER_LISTINGS"
+    );
+    await addDoc(commCollectionRef, {
+      formData,
+    }).then(() => {
+      setFormData(initialFormData);
+    });
+  };
+  
+  return (
+    <div>
+      <NavBar />
+      <h1 className={styles.firstBanner}>New Community</h1>
+      <form onSubmit={submitCommunity} action="/send-data-here" method="POST">
+        <container className={styles.communityLayout}>
+          {inputFields.map((field) => (
+            <div className={styles.layoutRow} key={field.name}>
+              <label htmlFor={field.name}>
+                {field.label}
+                {field.required && <span className="text-red-500">*</span>}
+              </label>
+              <input
+                type={field.type}
+                name={field.name}
+                id={field.name}
+                value={formData[field.name]}
+                onChange={handleChange}
+                required={field.required}
+                className={styles.inputStyle}
+                placeholder={field.placeholder}
+                pattern={field.pattern}
+              />
+            </div>
+          ))}
+        </container>
 
-                </div>
-                
+        <section className={styles.imageUploadBackground}>
```

Comment:
I think this could be a separate component that can be imported here. ImageUploader component.

Reply:
Removed due to descoping.